README.md

```diff
@@ -69,8 +69,13 @@ Choose a backend for the authoritative server. Valid values are 'mysql'. Default
 ##### `backend_install`
 
 If you set this to true it will try to install a database backend for
-you. This requires `db_root_password`, `db_username`, `db_password`,
```

Comment:
`db_username`, `db_password` etc. are also required when `backend_install` is set to false.

Reply:
I've got one more PR to create that adds initial postgresql support.  Perhaps I can add some advice about how `db_username` and `db_password` are always required (with either backend) there?

Reply:
I'll add it in a separate commit. No need to make it part of a new branch/pull I think. As usual thanks for the work, I'll merge this into develop.